src/Blocks/BlockGlazedTerracotta.h

```diff
@@ -0,0 +1,19 @@
+
+#pragma once
+
+#include "BlockHandler.h"
+#include "Mixins.h"
+
+
+class cBlockGlazedTerracottaHandler:
+	public cClearMetaOnDrop<cYawRotator<cBlockHandler, 0x3, 0x1, 0x3, 0x0, 0x2>>
+{
+	using super = cClearMetaOnDrop<cYawRotator<cBlockHandler, 0x3, 0x1, 0x3, 0x0, 0x2>>;
+
+public:
+
+	cBlockGlazedTerracottaHandler(BLOCKTYPE a_BlockType) : super(a_BlockType)
+	{
+	}
+
+};
```

Comment:
Couldn't this entire thing be replaced with a simple `using cBlockGlazedTerracottaHandler = cClearMetaOnDrop<cYawRotator<cBlockHandler, 0x3, 0x1, 0x3, 0x0, 0x2>>` ? If so, would it make sense to put that into the BlockHandler.cpp file? (this is a real question, not a suggestion as such)

Reply:
I think it can, there's not much point having an empty class really! It just turned out this way because I created this file before starting out the refactoring.